modules/oadp-installing-dpa-1-2-and-earlier.adoc

```diff
@@ -83,8 +83,7 @@ spec:
           key: cloud
           name: {credentials} <7>
   snapshotLocations: <8>
```

Comment:
Hi @anarnold97 . I see that [Installing the Data Protection Application 1.3](https://72144--ocpdocs-pr.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws#oadp-installing-dpa-1-3_installing-oadp-aws) section has a name entry in the YAML. OK?

Reply:
Yes